pkg/config/template_test.go

```diff
@@ -0,0 +1,111 @@
+package config
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"testing"
+
+	templateapi "github.com/openshift/api/template/v1"
+	templatescheme "github.com/openshift/client-go/template/clientset/versioned/scheme"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/diff"
+	"k8s.io/client-go/kubernetes/typed/core/v1/fake"
+	coretesting "k8s.io/client-go/testing"
+)
+
+const templatesPath = "../../test/pj-rehearse-integration/master/ci-operator/templates"
+
+func TestGetTemplates(t *testing.T) {
+	expectCiTemplates := getBaseCiTemplates(t)
+	if templates, err := getTemplates(templatesPath); err != nil {
+		t.Fatal(err)
```

Comment:
This err and the following one don't have enough context for someone who accidentally breaks the test